master/pkg/aproto/master_message.go

```diff
@@ -2,11 +2,10 @@ package aproto
 
 import (
 	"fmt"
-	"net"
 	"strconv"
 	"time"
 
-	"github.com/pkg/errors"
+	"golang.org/x/exp/maps"
```

Comment:
create a ticket for banning ```"golang.org/x/exp/maps"``` in favor of ```maps```?

also slices?

I don't remember what we said when we talked about this before

Reply:
maps doesn't have eveything x/exp/maps has yet.